integrationtests/src/test/java/io/kroxylicious/proxy/ExpositionIT.java

```diff
@@ -0,0 +1,270 @@
+                    CommonClientConfigs.SECURITY_PROTOCOL_CONFIG, SecurityProtocol.SSL.name,
+                    SslConfigs.SSL_TRUSTSTORE_LOCATION_CONFIG, clientTrustStore.toAbsolutePath().toString(),
+                    SslConfigs.SSL_TRUSTSTORE_PASSWORD_CONFIG, brokerCertificateGenerator.getPassword()))) {
+                // do some work to ensure connection is opened
+                admin.createTopics(List.of(new NewTopic(TOPIC, 1, (short) 1))).all().get();
+                var connectionsMetric = admin.metrics().entrySet().stream().filter(metricNameEntry -> "connections".equals(metricNameEntry.getKey().name()))
+                        .findFirst();
+                assertTrue(connectionsMetric.isPresent());
+                var protocol = connectionsMetric.get().getKey().tags().get("protocol");
+                assertThat(protocol).startsWith("TLS");
+            }
+        }
+    }
+
+    @Test
+    public void exposesTwoClusterOverPlainWithSeparatePorts(KafkaCluster cluster) throws Exception {
+        List<String> clusterProxyAddresses = List.of("localhost:9192", "localhost:9193");
+
+        var builder = KroxyConfig.builder()
+                .addNewFilter().withType("ApiVersions").endFilter()
+                .addNewFilter().withType("BrokerAddress").endFilter();
+
+        var base = new VirtualClusterBuilder()
+                .withNewTargetCluster()
+                .withBootstrapServers(cluster.getBootstrapServers())
+                .endTargetCluster()
+                .build();
+
+        for (int i = 0; i < clusterProxyAddresses.size(); i++) {
+            var virtualCluster = new VirtualClusterBuilder(base)
+                    .withNewClusterEndpointConfigProvider()
+                    .withType("StaticCluster")
+                    .withConfig(Map.of("bootstrapAddress", clusterProxyAddresses.get(i)))
+                    .endClusterEndpointConfigProvider()
+                    .build();
+            builder.addToVirtualClusters("cluster" + i, virtualCluster);
+
+        }
+        var config = builder.build().toYaml();
+
+        try (var proxy = startProxy(config)) {
+            for (int i = 0; i < clusterProxyAddresses.size(); i++) {
+                try (var admin = Admin.create(Map.of(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, clusterProxyAddresses.get(i)))) {
+                    // do some work to ensure virtual cluster is operational
+                    try {
+                        admin.createTopics(List.of(new NewTopic(TOPIC + i, 1, (short) 1))).all().get();
+                    }
+                    catch (Exception e) {
+                        throw new RuntimeException(e);
+                    }
+                }
+            }
+        }
+    }
+
+    @Test
+    public void exposesTwoClusterOverTlsWithSharedPort(KafkaCluster cluster) throws Exception {
+        var clientTrust = new ArrayList<ClientTrust>();
+        var virtualClusterCommonNamePattern = ".virtualcluster%d.multitenant.kafka";
```

Comment:
This isn't a multitenant test, so the multitenant  part of the domain name makes no sense.
I think we should be using addresses like `.kroxylicious.test` so adhere to  [RFC 2606 ](https://en.wikipedia.org/wiki/.test)

Reply:
Done